### PR TITLE
Issue/409 digest form

### PR DESCRIPTION
**Pull request by vedmant**
#409

### CODE REVIEW

**Comment by vedmant** [comment]
@aldarund how can I see what wercker/tests failed?

**Comment by aldarund** [comment]
@vedmant register at wercker and give me your mail that u used

**Comment by vedmant** [comment]
@aldarund Done, sent email in slack

File: back/appsync/functions/utils/models.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import os
```
Comment (aldarund) [inline back/appsync/functions/utils/models.py:1]: why its all in appsync while we  have other mail like daily mail in mail processing?

**Pull request merged**